docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,69 +1,269 @@
+
+endif::[]
+--
+
+. Pull the {kib} Docker image.
 +
+--
 ifeval::["{release-state}"=="unreleased"]
-NOTE: Version {version} of {es} has not been released,
-so the sample compose file is not yet available for this version.
-See the {stack-gs-current}/get-started-docker.html[current version] for the latest sample files.
+
+WARNING: Version {version} of {kib} has not yet been released, so no
+Docker image is currently available for this version.
+
 endif::[]
+
 ifeval::["{release-state}"!="unreleased"]
-["source","yaml",subs="attributes"]
---------------------------------------------
-include::docker/docker-compose.yml[]
---------------------------------------------
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/kibana/kibana:{version}
+----
+
+endif::[]
+--
+
+[[docker-start-enroll]]
+[discrete]
+==== Start a single-node {es} cluster and enroll {kib}
+
+The following command starts a single-node {es} cluster for development or
+testing.
+
+. Start {es} in Docker. A password is generated for the `elastic` user and
+output to the terminal, plus an enrollment token for enrolling {kib}.
++
+--
+ifeval::["{release-state}"=="unreleased"]
+
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+[source,sh,subs="attributes"]
+----
+docker run --name es-node01 --net elastic -p 9200:9200 -it docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
 endif::[]
-// tag::docker-memory[]
-. Make sure Docker Engine is allotted at least 4GiB of memory.
-In Docker Desktop, you configure resource usage on the Advanced tab in Preference (macOS)
-or Settings (Windows).
-// end::docker-memory[]
+--
++
+TIP: You might need to scroll back a bit in the terminal to view the password
+and enrollment token.
+
+. Copy the generated password and enrollment token and save them in a secure
+location. These values are shown only when you start {es} for the first time.
++
+[NOTE]
+====
+If you need to reset the password for the `elastic` user or other
+built-in users, run the {ref}/reset-password.html[`elasticsearch-reset-password`]
+tool. This tool is available in the {es} `bin` directory of the Docker container.
+
+For example, to reset the password for the `elastic` user:
+
+[source,sh]
+----
+docker exec -it es-node01 /usr/share/elasticsearch/bin/elasticsearch-reset-password -u elastic
+----
+====
 
-. Run `docker-compose` to bring up the three-node {es} cluster and {kib}:
+. Copy the `http_ca.crt` security certificate from your Docker container to
+your local machine. 
 +
-["source","sh",subs="attributes"]
+[source,sh]
+----
+docker cp es-node01:/usr/share/elasticsearch/config/tls_auto_config_<timestamp>/http_ca.crt .
+----
+`<timestamp>`:: The timestamp of when the auto-configuration process created
+the security files directory in your Docker container. Use the Docker CLI to
+locate the timestamp for this directory.
```

Comment:
We will see about changing this to be a standard dir name, but for now we can add the command for the user's reference:
```
docker exec -ti es-node01 /bin/bash -c "find /usr/share/elasticsearch -name http_ca.crt"
``

Reply:
Fancy! Thanks for this command; I'll definitely incorporate it into the workflow.